### ISSUE TITLE
Update flags to non legacy

### ISSUE THREAD

**Issue by birdsarah** [mozilla/openwpm-crawler#40]
when running `./start_gke_cluster.sh` I get the following warnings:

```
WARNING: From 1.14, legacy Stackdriver GKE logging is deprecated. Thus, flag `--enable-cloud-logging` is also deprecated. Please use `--enable-stackdriver-kubernetes` instead, to migrate to new Stackdriver Kuber
netes Engine monitoring and logging. For more details, please read: https://cloud.google.com/monitoring/kubernetes-engine/migration.
WARNING: From 1.14, legacy Stackdriver GKE monitoring is deprecated. Thus, flag `--enable-cloud-monitoring` is also deprecated. Please use `--enable-stackdriver-kubernetes` instead, to migrate to new Stackdriver
 Kubernetes Engine monitoring and logging. For more details, please read: https://cloud.google.com/monitoring/kubernetes-engine/migration.
```

**Comment by vringar** [comment]
Do you want to file a PR for this? I saw them as well but even with these clear instructions I don't wanna meddle around with the kubernetes stuff.

**Comment by birdsarah** [comment]
i'm not overly motivated to do it. anybody can when they've got a moment.

**Comment by vringar** [comment]
I fixed this in [@22cab47](https://github.com/mozilla/openwpm-crawler/commit/22cab47d9bc83bd0b6a914b92b30272adb2a24af)
